nginx.conf

```diff
@@ -0,0 +1,18 @@
+worker_processes 1;
+daemon off;
+
+error_log stderr;
+events { worker_connections 1024; }
+
+error_log /dev/stderr;
+
+http {
+  server {
+    listen 80;
+    server_name localhost;
+    return 503;
+  }
+
+  # Include Nginx errorpages configuration
+  error_page 503 /503.html;
```

Comment:
I don't think `error_page` is legal outside of a `server` block.  So I think you'll want something like

```
server {
  listen 80 default_server;

  error_page 503 /503.html;

  location / {
    return 503;
  }

  location = /503.html {
    root public;
  }
}
```

With `503.html` moved to `public/504.html` - I don't know if the `root` can be avoided.

Reply:
To also return something for the healthcheck endpoint we might need to hard-code the json:

```
  location = /healthcheck {
    default_type application/json;
    return 200 'json goes here';
  }
```

Reply:
I think we'll also want `try_files` for the `/` location, so we can serve assets:

```
 location / {
    try_files $uri =503;
  }
```